packages/toolkit/src/listenerMiddleware/index.ts

```diff
@@ -279,8 +281,16 @@ const defaultErrorHandler: ListenerErrorHandler = (...args: unknown[]) => {
   console.error(`${alm}/error`, ...args)
 }
 
+function cancelActiveListeners(
```

Comment:
```suggestion
const cancelActiveListeners = (
```

Reply:
Definitely a minor change but we save few bytes. 

Note that you also need to hoist this function above `createClearListenerMiddleware` if you want to accept this suggestion.